vaadinfrontend/src/main/java/life/qbic/datamanager/views/components/CardLayout.java

```diff
@@ -0,0 +1,243 @@
+   */
+  public void removeTitle() {
+    titleLayout.removeAll();
+  }
+
+  /**
+   * Adds custom css styling to the ButtonLayout within the CardLayout
+   *
+   * @param buttonStyles String representations of vaadin css classNames that will be added to the
+   *                     buttonLayout
+   */
+  public void addButtonStyles(String... buttonStyles) {
+    buttonLayout.addClassNames(buttonStyles);
+  }
+
+  /**
+   * Adds custom css styling to the fieldLayout within the CardLayout
+   *
+   * @param fieldStyles String representations of vaadin css classNames that will be added to the
+   *                    fieldLayout
+   */
+  public void addFieldStyles(String... fieldStyles) {
+    fieldLayout.addClassNames(fieldStyles);
+  }
+
+  /**
+   * Adds custom css styling to the fieldLayout within the CardLayout
+   *
+   * @param titleStyles String representations of vaadin css classNames that will be removed from
+   *                    the titleLayout
+   */
+  public void addTitleStyles(String... titleStyles) {
+    titleLayout.addClassNames(titleStyles);
+  }
+
+  /**
+   * Removes specified css styling of the buttonLayout within the CardLayout
+   *
+   * @param buttonStyles String representations of vaadin css classNames that will be removed from
+   *                     the buttonLayout
+   */
+  public void removeButtonStyles(String... buttonStyles) {
+    buttonLayout.removeClassNames(buttonStyles);
+  }
+
+  /**
+   * Removes specified css styling of the fieldLayout within the CardLayout
+   *
+   * @param fieldStyles String representations of vaadin css classNames that will be removed from
+   *                    the fieldLayout
+   */
+  public void removeFieldStyles(String... fieldStyles) {
+    fieldLayout.removeClassNames(fieldStyles);
+  }
+
+  /**
+   * Removes specified css styling of the titleLayout within the CardLayout
+   *
+   * @param titleStyles String representations of vaadin css classNames that will be added to the
+   *                    titleLayout
+   */
+  public void removeTitleStyles(String... titleStyles) {
+    titleLayout.removeClassNames(titleStyles);
+  }
+
+  /**
+   * Removes all styling of the buttonLayout within the CardLayout
+   */
+  public void removeAllButtonStyles() {
+    String[] currentButtonLayoutStyles = buttonLayout.getClassNames().toArray(String[]::new);
+    buttonLayout.removeClassNames(currentButtonLayoutStyles);
+  }
+
+  /**
+   * Removes all styling of the fieldLayout within the CardLayout
+   */
+  public void removeAllFieldStyles() {
+    String[] currentFieldLayoutStyles = fieldLayout.getClassNames().toArray(String[]::new);
+    fieldLayout.removeClassNames(currentFieldLayoutStyles);
+  }
+
+  /**
+   * Removes all styling of the titleLayout within the CardLayout
+   */
+  public void removeAllTitleStyles() {
+    String[] currentTitleLayoutStyles = titleLayout.getClassNames().toArray(String[]::new);
+    titleLayout.removeClassNames(currentTitleLayoutStyles);
+  }
```

Comment:
What does this public style handling do? It is not used and I do not understand the API. Which buttons are styled in which way?

Reply:
Hi @KochTobi I think this is what steffen mentioned in the PR description. It allows to customize the style of the card and e.g remove styles applied to the sublayouts. E.g the title, the buttons or the content (called field layout here)